repos/system_upgrade/el7toel8/libraries/config/version.py

```diff
@@ -0,0 +1,111 @@
+import operator
+
+import six
+
+from leapp.libraries.stdlib import api
+
+
+OP_MAP = {
+    '>': operator.gt,
+    '>=': operator.ge,
+    '<': operator.lt,
+    '<=': operator.le
+}
+
+
+def _version_to_tuple(version):
+    """Converts the version string ``major.minor`` to ``(major, minor)`` int tuple."""
+    major, minor = version.split('.')
+    return (int(major), int(minor))
```

Comment:
Just from the point that maybe in future there will be even X.Y.Z versioning used (maybe not, maybe yes), it would be better to write the whole stuff like:
```
    return tuple(int(x) for x in version.split('.'))
```